code/datums/uplink/gear loadout.dm

```diff
@@ -9,7 +9,7 @@
 /datum/uplink_item/item/gear_loadout/coalition_single
 	name = "Coalition of Colonies Assets (Single)"
 	path = /obj/structure/closet/crate/secure/gear_loadout/coalition_single
-	item_cost = 15
+	item_cost = 5
```

Comment:
Same here.
The number of crates a single person can buy with their TCs needs to be limited to 1.
(Or a system needs to be put in place that limits that otherwise)

Reply:
TC cost set to 10 for now. Might consider doing redeemable mechanics at a later date.